src/router/RouterProvider.jsx

```diff
@@ -18,7 +19,7 @@ const router = createBrowserRouter([
     children: [
       {
         index: true,
-        element: <AddEventPage/>,
+        element: <UpdateEvent/>,
```

Comment:
here is the home page why it is still the update event page here
after testing replace it with home page component